src/main/java/com/icthh/xm/gate/service/MonitoringService.java

```diff
@@ -81,6 +82,7 @@ public MonitoringService(ConsulClient consulClient,
      * @param serviceName name of the service
      * @return list of service health for each instance
      */
+    @SuppressWarnings("unchecked")
```

Comment:
Why we need this SuppressWarnings?

Reply:
Map<String, Object> healthStatus
healthStatus = new ObjectMapper().readValue(response.body().asInputStream(), Map.class);
this line produce unchecked warning

we can change Map<String, Object> to Map for avoding this uncheked cast